data/json/professions.json

```
@@ -4564,5 +4577,111 @@
         "entries": [ { "item": "bikini_top", "variant": "flag_bikini_top" }, { "item": "bikini_bottom", "variant": "flag_bikini_bottom" } ]
       }
     }
+  },
+  {
+    "type": "profession",
+    "id": "hitman",
+    "name": { "male": "Hitman", "female": "Hitwoman" },
+    "description": "You were the best undercover agent they had, with a mission so vital that you had to take it even with the riots going on.  Your target seems to have turned feral as well as your contact, you are all alone now.",
+    "points": 5,
+    "skills": [
+      { "level": 3, "name": "dodge" },
+      { "level": 1, "name": "melee" },
+      { "level": 3, "name": "gun" },
+      { "level": 3, "name": "pistol" }
+    ],
+    "proficiencies": [ "prof_gunsmithing_basic", "prof_spotting", "prof_wp_syn_armored" ],
+    "items": {
+      "both": {
+        "items": [
+          "fancy_sunglasses",
+          "platinum_watch",
+          "knit_scarf_loose",
+          "gloves_liner",
+          "polaroid_photo",
+          "suppressor",
+          "tele_sight_pistol"
+        ],
+        "entries": [
+          { "item": "glock_29", "ammo-item": "10mm_fmj", "charges": 15 },
```

```suggestion
          "polaroid_photo"
        ],
        "entries": [
          { "item": "glock_29", "ammo-item": "10mm_fmj", "charges": 15, "contents-item": [ "tele_sight_pistol", "suppressor" ] },
```
So it spawns attached, like Military Recruit's strap and holographic sight are

---

It's not possible, the holster is small so the pistol does not fit with the mods attached (I tried hahaha). I though of increasing the holster pocket, but I'm not sure about that.